Specification/5._Application_level_conventions.md

```diff
@@ -56,7 +54,7 @@ In the case of cameras, the following convention should be preferred:
 
 For world frames, the North-East-Down (NED) notation should be preferred.
```

Comment:
Link to wikipedia article on NED?

Reply:
Sure, I think it should be safe to link Wikipedia. Could you please add the link as I can't push to your fork myself?